helm/values-dev.yaml

```diff
@@ -31,6 +31,15 @@ microservice-chart:
     ENV: "DEV"
     JAVA_TOOL_OPTIONS: "-Xms128m -Xmx4g -Djava.util.concurrent.ForkJoinPool.common.parallelism=7 -javaagent:/app/applicationinsights-agent.jar -Dapplicationinsights.configuration.file=/mnt/file-config-external/appinsights-config/applicationinsights.json -agentlib:jdwp=transport=dt_socket,server=y,address=8001,suspend=n -Dcom.sun.management.jmxremote=true -Dcom.sun.management.jmxremote.port=3002 -Dcom.sun.management.jmxremote.rmi.port=3003 -Djava.rmi.server.hostname=127.0.0.1 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false"
 
+    PDND_BASE_URL: https://auth.uat.interop.pagopa.it
+    PDND_ACCESS_TOKEN_AUDIENCE: auth.uat.interop.pagopa.it/client-assertion
+
+    PAGOPA_PDND_CONFIGURATION_CLIENT_ID: 890b7ca9-b402-4dce-9e8d-9a333d22d76d
+    PAGOPA_PDND_CONFIGURATION_KID: jxOpPRxM6oFcnnKtICqeW5l7fbxLr45IAsJ8Q9s-fK8
+    PAGOPA_PDND_CONFIGURATION_PURPOSE_ID: 87520bd5-207a-4616-85d9-10d7bb3e88b8
+    PDND_PRIVATE_KEY: piattaforma-unitaria-interop-priv
+    PDND_PUBLIC_KEY: piattaforma-unitaria-interop-pub
```

Comment:
are these secrets? if yes, this is the wrong section, they should be in the values.yml under the envSecret key